python/lsst/daf/butler/core/registry.py

```diff
@@ -115,5 +116,7 @@ def fromConfig(registryConfig, schemaConfig=None, create=False):
     def __init__(self, registryConfig, schemaConfig=None, create=False):
         assert isinstance(registryConfig, RegistryConfig)
         self.config = registryConfig
+        pixelizationCls = doImport(self.config["skypix.cls"])
```

Comment:
Make `self.pixelization` a property so that import is deferred until it's actually needed?